cheat/app/src/main/java/at/tugraz/ist/sw20/swta1/cheat/ui/chat/ChatEntry.kt

```diff
@@ -20,6 +20,13 @@ class ChatEntry(private var message: String, var isByMe: Boolean, var isBySystem
         return message
     }
 
+    fun getMessageShortened(): String {
+        if(message.length > 43) {
```

Comment:
shout we put magic numbers in CONSTANTS?

Reply:
fixed